File: Beginner/Lost Weekends (LOSTWKND)/weekends.py
t = int(input())


while t:
    A1 , A2, A3, A4, A5 , P=map(int,input().split())
    s = (A1 + A2 + A3 + A4 + A5)*P
    C = "No"
    if s>120:
        C ="Yes"
    print(C)

    t = t-1

File: Beginner/Lost Weekends (LOSTWKND)/weekends.cpp
#include <bits/stdc++.h>
using namespace std;

int main() {
	int t;
	cin>>t;
	while(t-->0) {
        int A[5];
        for(int i=1;i<=5;i++) {
            cin>>A[i];
        }
        int P;
        cin>>P;
        int s=0;
        for(int i=1;i<=5;i++) {
            s += A[i]*P;
        }
        if(s<=120) {
            cout<<"No"<<endl;      
        }
        else {
            cout<<"Yes"<<endl;
        }
        
	}
	return 0;
}
